src/modules/mavlink/streams/BATTERY_INFO.hpp

```diff
@@ -67,36 +67,49 @@ class MavlinkStreamSmartBatteryInfo : public MavlinkStream
 
 			if (battery_sub.update(&battery_status)) {
 				if (battery_status.serial_number == 0) {
-					// This is not smart battery
+					// Required to emit
```

Comment:
We're no longer identifying as "smart batteries". This might not be a reasonable thing to check on.